### PR TITLE
gradient background needs GL_MODELVIEW

### PR DESCRIPTION
**Pull request by MichaelPFrey**
the gradient back only worked properly with show axis enabled.

### CODE REVIEW

File: src/glview/GLView.cc [hunk 1]
```diff
@@ -131,9 +131,13 @@ void GLView::paintGL()
 
   if(bgcol != bgstopcol){
     glDisable(GL_DEPTH_TEST);
+
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
 
+    glMatrixMode(GL_MODELVIEW);
+    glLoadIdentity();
```
Comment (t-paul) [inline src/glview/GLView.cc:139]: Is the `GL_PROJECTION` part still needed? It looks to me like that would do nothing?

**Review by t-paul — CHANGES_REQUESTED** [review]

File: src/glview/GLView.cc (hunk 1 above)
Comment (MichaelPFrey) [reply src/glview/GLView.cc:139]: when I comment out `glMatrixMode(GL_PROJECTION);`, there is a square centered on the screen, which size depends on zoom level.

I followed https://community.khronos.org/t/gradient-background/54348

The rough idea seams to be to setup openGL that the corners of the view are a 1 unit away from center (independent of aspect ratio of the view), then look straight on, then look onto a square with gradient.

Comment (thehans) [reply src/glview/GLView.cc:139]: Both ModelView and Projection matrices are combined for the overall transformation (sometimes referred to as MVP).  Loading the identity resets whichever matrix is currently specified by `glMatrixMode`, but both are used simultaneously on all vertices going through the fixed function pipeline.

Comment (t-paul) [reply src/glview/GLView.cc:139]: Ah, ok. Thanks.

**Review by t-paul — APPROVED** [review]

**Pull request merged**